index.html

```diff
@@ -5,83 +5,80 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <link rel="stylesheet" href="./assets/css/style.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" integrity="sha512-UJfAaOlIRtdR+0P6C3KUoTDAxVTuy3lnSXLyLKlHYJlcSU8Juge/mjeaxDNMlw9LgeIotgz5FP8eUQPhX1q10A==" crossorigin="anonymous" />
     ​<script src="https://code.jquery.com/jquery-3.5.1.min.js"   integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="   crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <title>Daily-Dashboard</title>
 </head>
 
-<body> ​
+<body> 
     <header>
-        <h2>Welcome back, Amy!</h2>
-        <h3>Check out today's weather and news</h3>
-        <h1>Amy Legge</h1>
+
+        <h2>Daily Dashboard</h2>
+
+        <h4>Check out today's weather and news</h4>
+        
+        <p><button id="modal_open">Welcome!</button></p>
+
+        <div id="modal_wrapper">
+        <div id="modal_window">
+
+            <div style="text-align: right;"><a id="modal_close" href="#">close <b>X</b></a></div>
+
+            <p>Complete the form below:</p>
+
+            <form id="modal_form" method="POST" action="#" accept-charset="UTF-8">
+            <p><label>Your Name<strong>*</strong><br>
+            <input type="text" autofocus required size="48" name="name" value=""></label></p>
+            <p><label>Your City<strong>*</strong><br>
+            <input type="text" required title="Please enter your city" size="48" name="city" value=""></label></p>
+            <p><input type="submit" name="modal_form" value="Submit"></p>
+            </form>
+
+            </div> <!-- #modal_window -->
+            </div> <!-- #modal_wrapper -->
+
     </header>
     ​
-    <section class="weather wrapper">
```

Comment:
Amy's code may be referencing the class "weather-wrapper" and may break due to this change. Please take note, Amy.